cinema/serializers.py

```diff
@@ -22,3 +22,60 @@ def update(self, instance, validated_data):
         instance.save()
 
         return instance
+
+
+class CinemaHallSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    name = serializers.CharField(max_length=63, required=True)
+    rows = serializers.IntegerField(required=True)
+    seats_in_row = serializers.IntegerField(required=True)
+
+    def create(self, validated_data):
+        return CinemaHall.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.name = validated_data.get("name", instance.name)
+        instance.rows = validated_data.get(
+            "rows", instance.rows
+        )
+        instance.seats_in_row = validated_data.get(
+            "seats_in_row", instance.seats_in_row
+        )
+
+        instance.save()
+
+        return instance
+
+
+class GenreSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    name = serializers.CharField(max_length=63, required=True)
+
+    def create(self, validated_data):
+        return Genre.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.name = validated_data.get("name", instance.name)
+
+        instance.save()
+
+        return instance
+
+
+class ActorSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    first_name = serializers.CharField(max_length=63, required=True)
+    last_name = serializers.CharField(max_length=63, required=True)
```

Comment:
Any other fields here or in movies?)

Reply:
It's been noticed in Readme, that we must not use related serializers for ManyToMany relations. For genres and actors there are ManyToMany relations and if I use extra field for Genre or Actor Serializer, I need to implement another Serializer for list or retrieve action, but we have no such a task here. Please, advice, what fields should i add there?